### DISCUSSION TITLE
Custom postgresql.conf file with docker-compose

### DISCUSSION THREAD

**Discussion by fuzunspm** [kartoza/docker-postgis#464] [Q&A] [answered]
Hi,

I'm not sure how should i edit or pass custom postgresql.conf file with docker-compose?

**Comment by NyakudyaA** [comment, marked as answer]
You can define all the settings in a conf like

```
docker run --name "postgis" -v /data/extra.conf:/settings/extra.conf -p 25432:5432 -d -t kartoza/postgis
```
Anything in extra.conf will take precedence
